<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieDialog</class>
 <widget class="QDialog" name="MovieDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="validationButtons">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>340</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>205</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="titleEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="originalTitleLabel">
      <property name="text">
       <string>Original Title</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="originalTitleEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="releaseDateLabel">
      <property name="text">
       <string>Release Date</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="countryLabel">
      <property name="text">
       <string>Country</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="countryEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="synopsisLabel">
      <property name="text">
       <string>Synopsis</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPlainTextEdit" name="synopsisEdit"/>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QDateEdit" name="releaseDateEdit">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>1900</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>1800</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="minimumDate">
         <date>
          <year>1800</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="date">
         <date>
          <year>1900</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetReleaseDateBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>371</width>
     <height>313</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="3">
     <widget class="QPushButton" name="addProducerButton">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QListWidget" name="producersWidget">
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QListWidget" name="directorsWidget"/>
    </item>
    <item row="3" column="3">
     <layout class="QVBoxLayout" name="producersBtn"/>
    </item>
    <item row="1" column="3">
     <layout class="QVBoxLayout" name="directorBtns"/>
    </item>
    <item row="5" column="2">
     <widget class="QListWidget" name="actorsWidget"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="addDirectorButton">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="directorEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="actorsLabel">
      <property name="text">
       <string>Actors</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="directorsLabel">
      <property name="text">
       <string>Director(s)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="producersLabel">
      <property name="text">
       <string>Producer(s)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="producerEdit"/>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="addActorButton">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="actorEdit"/>
    </item>
    <item row="4" column="4">
     <layout class="QVBoxLayout" name="actorsBtn">
      <item>
       <widget class="QPushButton" name="delActorButton">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="delProducerButton">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="delDirectorButton">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>237</y>
     <width>421</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>Add a tag here</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="newTagLabel">
          <property name="text">
           <string>New tag</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="newTagLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="addNewTagButton">
        <property name="text">
         <string>add tag</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Tag list, select the tags for this movie.  
Multi-selection with &lt;Ctrl&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="tagListView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>validationButtons</sender>
   <signal>accepted()</signal>
   <receiver>MovieDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>validationButtons</sender>
   <signal>rejected()</signal>
   <receiver>MovieDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
